server/src/main/java/de/hellfish/ngrok/server/ClientConnectionService.java

```diff
@@ -0,0 +1,13 @@
+package de.hellfish.ngrok.server;
+
+import java.net.Socket;
+
+public interface ClientConnectionService {
```

Comment:
Why an interface is created?

Reply:
To make realization and definition separate. Code looks better, easier to support. App is more flexible and finally support better polymorphism

Reply:
But finally i red this article 

https://medium.com/w-logs/think-twice-before-implementing-interface-classes-in-microservices-abc67ae11fac

and i should of course remove interface here.